### ISSUE TITLE
Copy to/Paste from Clipboard

### ISSUE THREAD

**Issue by mowgli** [BestImageViewer/geeqie#261]
I've been looking around a little bit for an image viewer that can handle copying/pasting an image to/from the clipboard.  It doesn't seem to be present in any of the image viewing applications I've tried:
Eye of Gnome
GQvieq /GeeQie
Mirage
GPicView
GwenView
GThumb

Is there a reason for this?  I found it quite surprising, it seems like such an obvious feature to have (maybe thats my old windows experience talking).  But it also seems a little strange because most of these applications use gtk, which seems to have easy clipboard support (I wrote a pygtk program that pastes an image from the clipboard). 

In short I guess I'm asking for clipboard support :)

Reported by: *anonymous

**Comment by caclark** [comment]
This can be achieved easily enough with a desktop file and xclip.
I suppose it is not used so frequently that it it should be integrated into Geeqie itself.
